pinery-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoBoxPosition.java

```diff
@@ -0,0 +1,84 @@
+package ca.on.oicr.pinery.lims.miso;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import ca.on.oicr.pinery.api.Box;
+import ca.on.oicr.pinery.api.BoxPosition;
+import ca.on.oicr.pinery.lims.DefaultBox;
+import ca.on.oicr.pinery.lims.DefaultBoxPosition;
+
+public class MisoBoxPosition extends DefaultBox implements BoxPosition {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2160)